test/transformers/OtherAllowancesTransformerTest.scala

```diff
@@ -89,14 +91,10 @@ class OtherAllowancesTransformerTest extends BaseSpec with AtsJsonDataUpdate wit
 
       val data = Json.obj("ctnMarriageAllceOutAmt" -> Amount(200.00, "GBP"))
       val amendedJson = JsonUtil.loadAndReplace("/test_case_3.json", data)
+      val calculations = ATSCalculations.make(amendedJson.as[TaxSummaryLiability], taxRate)
 
       val returnValue: AtsMiddleTierData =
-        ATSRawDataTransformer(
-          applicationConfig,
-          amendedJson.as[TaxSummaryLiability],
-          parsedTaxpayerDetailsJson,
-          "",
-          taxYear).atsDataDTO
+        SUT.atsDataDTO(taxRate, calculations, parsedTaxpayerDetailsJson, "", taxYear)
 
       val parsedYear = returnValue.taxYear
       val testYear: Int = 2014
```

Comment:
Should we change the test year to one more relevant?

Reply:
No. new change will be completely independent including the tests so it can done after.